main.go

```diff
@@ -74,6 +93,8 @@ func main() {
 		slog.Infof("HTTP server error: %v", err)
 	case err := <-errsTLS:
 		slog.Infof("TLS server error: %v", err)
+	case err := <-errKN:
+		slog.Infof("failed to initialize crds: %v", err)
```

Comment:
I had to do the error checking at the bottom, otherwise it blocks the execution, is it okay for error checking and actual initialization to be seperate?

Should I just, remove returning of an error, and close the program `os.Exit(1)` when an error occurs in 	`kubenotation.Start`

Reply:
The library now does not return error, and instead logs the error and causes a `os.Exit(1)` in case of a failure: https://github.com/nirmata/kyverno-notation-aws/pull/17/commits/486244234848cb0e63feede5c1863bf62eb0bbd9